example/PercentageArea.purs

```diff
@@ -0,0 +1,304 @@
+module PercentageArea where
+
+import Prelude
+import Control.Monad.Eff.Console (print)
+import Data.Tuple.Nested
+import Data.Tuple
+import Data.Maybe
+import Data.Array (zipWith, head, tail, cons)
+import Data.Function
+import Utils
+
+import ECharts.Chart
+import ECharts.Options
+import ECharts.Tooltip
+import ECharts.Toolbox
+import ECharts.Coords
+import ECharts.Legend
+import ECharts.Axis
+import ECharts.Series
+import ECharts.Item.Data
+import ECharts.Item.Value
+import ECharts.Common
+import ECharts.Formatter
+import ECharts.Style.Item
+import ECharts.Style.Text
+import ECharts.Style.Line
+import ECharts.Style.Area
+import ECharts.Color
+import ECharts.Title
+import ECharts.Symbol
+import Control.Monad.Eff (Eff())
+
+simpleData = Value <<< Simple
+
+foreign import anotationFomatter :: forall eff. (Eff eff Unit)
+
+mkAnotationItem :: ItemValue -> ItemDataDatRec
+mkAnotationItem value =
+  {
+    value: value,
+    name: Nothing,
+    tooltip: Nothing,
+    itemStyle: 
+      Just $ ItemStyle itemStyleDefault {
+          normal = Just $ IStyle istyleDefault {
+            label = Just $ ItemLabel itemLabelDefault { 
+              show = Just $ true,
+              position = Just LPRight,
+              formatter = Just $ Template "\n{a}",
+              textStyle = Just $ TextStyle textStyleDefault {
+                fontFamily = Just "Palatino, Georgia, serif",
+                fontSize = Just 10.0,
+                color = Just "rgba(255,255,255,1)",
+                align = Just HAlignRight,
+                baseline = Just TBLBottom
+                }       
+              }  
+            }   
+          },
+    selected: Nothing
+  }
+
+fromJustArray :: forall a. Maybe (Array a) -> Array a
+fromJustArray arr = do
+  case arr of
+    Nothing -> []
+    Just arr ->  arr
+
+fromJustNumber :: Maybe Number -> Number
+fromJustNumber n = do
+  case n of
+    Nothing -> 0.0
+    Just n ->  n
+
+
+options :: Option
+options = Option $ optionDefault {
```

Comment:
FYI, this code format for records is kinda deprecated (not a big deal, but you may find new indentation a bit more convenient) this would be something like 
```purescript
options = 
  Option $ optionDefault 
    { title = Just $ Title blablabla... 
    , tooltip = Just $ Tooltip tooltipDefault 
        { trigger = Just Trigger Axis
        , textStyle = blablabla ...
        }
...

```
This allows add and remove fields w/o changing previous line :smile: